keras_nlp/__init__.py

```diff
@@ -19,6 +19,14 @@
 
 import os
 
+# sentencepiece is segfaulting on tf-nightly if tensorflow is imported first.
+# This is a temporary fix to restore our nightly testing to green, while we look
+# for a longer term solution.
```

Comment:
Does this comment still stand?

Reply:
i'm not sure if we still need this weird work around or not. i'll leave it in for now, try removing on a separate pr